0.Data_Preprocessing/notebooks/0.generate_toy_data.ipynb

```diff
@@ -0,0 +1,442 @@
oqOQDWIBRBYnPa8cOo0KOo1adBQiMiOZTIZbo3yx/WQpzpVzYkTqXSyAyOLsPM3lL4hsxZhQT2gc7PFROidGpN7FAogsSuHlelyorOftLyIbobSTIzHy6sSIlbUG0XHIhrAAIouyK68cchkQ1Y/LX1DnSQo7lN39G5Td/RsuhWGFpkbqAACf7M0XnIRsCT8pyKLsOl2OgToXOCr5vyZ1nqRS4ez//VN0DOomF7U9xg/yQnLGBTwxLoTL31CvYAsQWYzmn5a/GMrh70Q257YhvtA3NOPzrELRUchGCC+AVqxYgeDgYKjVasTExGD37t3XPba4uBgPPvggwsLCIJfLsWDBgg6P+/LLLxEZGQmVSoXIyEhs2LDBTOmpJ+UUVqHW0ML+P9R1kgR5fR3k9XVcCsNKeWvUGD3AA6t2nUMLl8egXiC0AEpNTcWCBQuwePFiZGdnY+zYsZgxYwYKCjpeIM9gMMDLywuLFy9GdHR0h8dkZmYiKSkJs2fPxuHDhzF79mzcf//92LdvnzkvhXrArtPlcFHbYYCnk+goZGXkDfWIGxqEuKFBXArDit0+1A8XrzRgy7ES0VHIBsgkgXOQx8XFYcSIEVi5cmXrvoiICMyaNQtLly694WsnTJiAYcOGYdmyZW32JyUlQa/XY8uWLa37pk+fDjc3N6SkpHQql16vh1arRXV1NTQadsbtLXe+lw4nlR1+P3mg6ChkZeT1dYgbGgQA2HckHyZHFtHW6u+bT8JokvDt72+BTMalcKhruvL9LawFqKmpCVlZWUhMTGyzPzExERkZGd0+b2ZmZrtzTps27YbnNBgM0Ov1bTbqXZfrmnC0qJqrvxPZuDuj/XCiWI9deRWio1AfJ6wAqqiogNFohE6na7Nfp9OhpKT7zZ8lJSVdPufSpUuh1Wpbt4CAgG7/fOqe3XnlkAD2/yGycYP9NAjxcsKKH8+IjkJ9nPBO0Nc2cUqSdNPNnl0956JFi1BdXd26FRZyFEJv25FbjmBPR7g7KUVHISKBZDIZZkb3w77zl5GVz0VSyXyEFUCenp5QKBTtWmbKysrateB0hY+PT5fPqVKpoNFo2mzUe0wmCTtyy9j6Q0QAgJj+bujn6oCVO9gKROYjrABSKpWIiYlBWlpam/1paWlISEjo9nnj4+PbnXPbtm03dU4yryNF1bhS34xhAa6ioxCRBZDLZLgj2g/bT5bhZDH7ZJJ5CJ1ud+HChZg9ezZiY2MRHx+PVatWoaCgAPPnzwdw9dZUUVER1qxZ0/qanJwcAEBtbS3Ky8uRk5MDpVKJyMhIAMCzzz6LcePG4bXXXsPMmTOxceNGbN++Henp6b1+fdQ5O3LL4KRSYKC3i+goZKUkhQKV0+9sfUzWb0yoB746dBHLfzyDfz44QnQc6oOEFkBJSUmorKzEkiVLUFxcjKioKGzevBlBQVeHsxYXF7ebE2j48OGtj7OysrB27VoEBQXhwoULAICEhASsW7cOL774Il566SWEhIQgNTUVcXFxvXZd1DU/nCrDkH5aKOQc8krdI6nUOP3Pj0XHoB5kJ5fj9qF+WL3nPJ6bWosQL2fRkaiPEToPkKXiPEC9p7LWgNi/bscT4wdg/CBv0XGIyII0tZjw3PocTAr3whv3DRMdh6yAVcwDRARcXf1dAhDNDtBEdA2lnRy3DfHFhkOXUHiZM3xTz2IBREL9cLIMIV5OcHXk8HfqPnl9HeJDPREf6nl1PTDqMyZHeMNZbYcVHBFGPYwFEAnTbDRhx+lyDAtwEx2FiCyUyk6B24f6Yv3Bi2wFoh7FAoiEycq/gprGFowIdBUdhYgs2JQIHZxVdlix46zoKNSHsAAiYX44VQY3R3v05+rvRHQDansFbh3ii88PFuLiFbYCUc9gAUTCbD9RimEBbpBzxWci+hWJkTo4KhVY/iNbgahnsAAiIS5U1OFcRR1vfxFRp6jtFbgj2g/rDxaioJKtQHTzWACRED+cKoO9QoaoflrRUYjISkyN1EGjtsO7P5wWHYX6ABZAJETaiVJE+mqgtueyBXTzJIUCVyZMwZUJU7gURh+mslPgzmg/fHWoCOfKa0XHISvHAoh6XXV9M/afv4yYIHfRUaiPkFRqnPpwHU59uA6SSi06DpnRpHAd3JyUeDuNrUB0c1gAUa/7IbcURklCTBDn/yGirlHayXHX8H745kgxjl+qFh2HrBgLIOp1W4+XItTbGe5OnP2ZiLpuwiBv+GrVeH1rrugoZMVYAFGvamw2YmduOWIC2fpDPUdeX4dRQwIxakggl8KwAQq5DPfF+GNHbjkOXLgsOg5ZKRZA1KsyzlagodmI2P4sgKhnKRrqoWjg8GhbETfAA8GeTvjHllOQJEl0HLJCLICoV6WdKIWvVo1+rg6ioxCRFZPLZEiKDUBW/hVsP1kmOg5ZIRZA1GuMJgnbTpQiJsgNMs7+TEQ3aai/FkP6abF0y0m0GE2i45CVYQFEvebghcuorG3CyP4c/k5EN08mk+GBUYE4V16H1IOFouOQlWEBRL1my7ESeDgpEertLDoKEfURwZ5OuCXUE2+lnUadoUV0HLIiLICoV5hMErYcK0Zsf3cufkpEPer+2ADoG5rx/k4ulEqdxwKIekV2YRVK9QbEBfP2F5mBXI7qUQmoHpUAyPmxZmu8XFS4dYgvVu06h6KqBtFxyErwk4J6xXfHiuHqaI8wnYvoKNQHmdQOOLF2E06s3QSTmiMMbdHM6H5wsFfg/7acEh2FrAQLIDI7SZLw7dFixAa5QS7n7S8i6nkOSgXujw3AxsOXcKjgiug4ZAVYAJHZHS2qxqWqRowK9hAdhYj6sPGDvNDfwxGvbDwOk4mTI9KNsQAis/vm8CVoHewR6asRHYX6KHl9HWJHhiF2ZBiXwrBhcrkMDyf0x5GianyRdVF0HLJwLIDIrEwmCd8cLkZcsDsUvP1FZmR/pRL2VypFxyDBwn00uCXUE//47hSqG5pFxyELxgKIzOpg/hWU6BuREOIp
```

Comment:
Depending on how you plan to use notebooks and `jupytext` with this project, consider adding the following to your `pyproject.toml` file in order to tell `jupytext` how to pair notebook files throughout the project. This might need to be modified in order to retain the "scripts" sub-directory structure.

```toml
[tool.jupytext]
formats = "ipynb,py:percent"
```

Reply:
notebook is an artifact. I have been using them for trouble shooting but I dont't think I will have any soon.